blocks/footer/footer.css

```diff
@@ -1,14 +1,230 @@
+/* stylelint-disable no-descending-specificity */
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
-  font-size: var(--body-font-size-s);
+  background-color: #27292b;
```

Comment:
all colors should be put into CSS vars, categorized into primary, secondary and so on colors.